src/backend/communities.nim

```diff
@@ -385,8 +385,12 @@ proc collectCommunityMetrics*(communityId: string, metricsType: int, intervals:
       "intervals": intervals
     }])
 
-proc requestCommunityInfo*(communityId: string): RpcResponse[JsonNode] {.raises: [Exception].} =
-  result = callPrivateRPC("requestCommunityInfoFromMailserver".prefix, %*[communityId])
+proc requestCommunityInfo*(communityId: string, useDatabase: bool): RpcResponse[JsonNode] {.raises: [Exception].} =
+  result = callPrivateRPC("requestCommunityInfoFromMailserverV2".prefix, %*[{
```

Comment:
FYI, there is a new `requestCommunityInfoFromMailserver` API function already that now accepts shard info. 
There is this issue to update to it https://github.com/status-im/status-desktop/issues/12015
So you might conflict. Maybe you can check in with @shinnok

Reply:
I just saw your status-go PR, it seems like you handled it, so there probably won't be a bug conflict :+1: